chore(release): Automated agent plugin release notes (v0.9.17) (#909)
armory/docs: content/en/docs/release-notes/rn-armory-agent/agent-plugin/agent-plug-v-0-9-17.md

---
title: v0.9.17 Armory Agent Clouddriver Plugin (2021-09-30)
toc_hide: true
version: 00.09.17

---

### Fixes

> - Kubernetes kinds not accessible anymore for operations when first attempt resulted in failure

Reviewer: ```suggestion
- Fixed an issue where Kubernetes kinds were not accessible for operations if the first attempt failed.
```
